assets/src/edit-story/components/panels/animation/effectInput.js

```diff
@@ -88,9 +91,21 @@ function EffectInput({
         </RangeContainer>
       );
     case FIELD_TYPES.DIRECTION_PICKER:
+      /**
+       * Zoom is implemented with two numeric values: zoomTo and zoomFrom. Visually it is displayed
+       * to the user as a scaleIn or scaleOut picker. This converts the numeric values
+       * to a string-base identifer for the visual picker.
+       */
+      if ('zoomFrom' in effectConfig && 'zoomTo' in effectConfig) {
+        valueForField =
+          effectConfig.zoomFrom < effectConfig.zoomTo
+            ? SCALE_DIRECTION.SCALE_OUT
+            : SCALE_DIRECTION.SCALE_IN;
+      }
```

Comment:
is this needed anymore with the update to the effect taking direction instead of from/to?